main.go

```diff
@@ -273,6 +276,18 @@ func (o *Options) Run() error { //nolint:gocyclo
 		}
 	}
 
+	timeNow := time.Now()
```

Comment:
Having this hard coded seems problematic to me, because the server will forever be pinned to the time you started it with.  You'll never get new data without restarting